cms/envs/common.py

```diff
@@ -2678,3 +2678,12 @@
 COURSE_LIVE_GLOBAL_CREDENTIALS = {}
 
 PERSONALIZED_RECOMMENDATION_COOKIE_NAME = 'edx-user-personalized-recommendation'
+
+######################## Registration ########################
+
+# Social-core setting that allows inactive users to be able to
+# log in. The only case it's used is when user registering a new account through the LMS.
+INACTIVE_USER_LOGIN = True
+
+# Redirect URL for inactive user. If not set, user will be redirected to /login after the login itself (loop)
+INACTIVE_USER_URL = f'http://{CMS_BASE}'
```

Comment:
I think in an earlier version of this PR you had something to switch to https:// if the `HTTPS` setting was enabled -- did that cause problems?

Reply:
Common settings have the `HTTPS == 'on'` 

Using the previous version I had either add changes to the devstack.py or move that check to the production.py I chose the second option.

Reply:
Ah! I see the change to production.py now.